inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseProfileDeserializationTest.java

```diff
@@ -55,17 +58,18 @@ public void checkThatDeserializationWorks() throws JsonParseException, JsonMappi
         String descriptionIri = "http://www.w3.org/2000/01/rdf-schema#comment";
         String propertyLabelIri = "http://www.w3.org/2000/01/rdf-schema#label";
         String propertyDescriptionIri = "http://www.w3.org/2000/01/rdf-schema#comment";
-
+        List<String> rootConcepts =  Arrays.asList(OWL.NOTHING.stringValue());
```

Comment:
Why do we still need the `OWL.NOTHING`? It should be sufficient if the list is empty. Saying explicitly that there is "no root concept" by adding `OWL.NOTHING` to the list is not required.